packages/salesforcedx-utils-vscode/src/cli/commandExecutor.ts

```diff
@@ -24,11 +26,26 @@ export interface CancellationToken {
 
 export class CliCommandExecutor {
   private readonly command: Command;
-  private readonly options: ExecOptions;
+  private readonly options: SpawnOptions;
```

Comment:
I made a mistake earlier and used ExecOptions instead of SpawnOptions. We should use SpawnOptions.
@guw I see that you duplicate this code in your ISV boostrap command. You have to change it yourself.

Reply:
@vazexqi Can you point to the duplication? Are you referring to the overriden `executeCommand` method? I think it would be good if such refactorings are captured in the same PR for traceability in code history, though.

Reply:
What was the reason for code duplication? Could you eliminate that duplication?

Reply:
I see where it is now. I can fix it in this PR.